package.json

```diff
@@ -12,7 +12,7 @@
     "start": "npm-run-all --parallel watch:**",
     "watch:es": "esbuild public/demo.ts --global-name=minidoc --bundle --servedir=public --outfile=public/dist/bundle.js --sourcemap --define:process.env.NODE_ENV=\"'development'\"",
     "watch:ts": "tsc --noEmit --watch --preserveWatchOutput",
-    "prepare": "rm -rf ./dist && npm test && npm-run-all --parallel release:**",
+    "prepack": "rm -rf ./dist && npm test && npm-run-all --parallel release:**",
```

Comment:
`prepare` was running on a fresh `npm install`. So changed it with `prepack` to prevent running the tests twice on the cli.